### DISCUSSION TITLE
New Docker Compose Options - home / working_dir

### DISCUSSION THREAD

**Discussion by tootbrute** [photoprism/photoprism#1950] [Q&A] [answered]
The default Docker Compose configuration file has a few more options than when I configured Photoprism a few months ago.

I had to re-install programs on my server, so I'm going to use my old Docker Compose file to load up my previous PhotoPrism installation but I'm curious if I should update/edit some of the options. I don't see these mentioned in the [Config Options.](https://docs.photoprism.app/getting-started/config-options/)

**New options**
HOME: "/photoprism" #what this?
working_dir: "/photoprism" #what's this?

image: mariadb:10.6 #changed from image: mariadb:10.5 before
There is also a bit of a new command string for mariadb. The buffer pool switch is new.
`command: mysqld **--innodb-buffer-pool-size=128M** --transaction-isolation=READ-COMMITTED --character-set-server=utf8mb4 --max-connections=512 --innodb-rollback-on-timeout=OFF --innodb-lock-wait-timeout=120`

Are the above changes "safe" to make?

**Comment by lastzero** [comment, marked as answer]
Yes, home is the default. For MariaDB, see https://docs.photoprism.app/getting-started/troubleshooting/performance/#mariadb
